scss/_close.scss

```diff
@@ -5,21 +5,21 @@
   line-height: 1;
   color: $close-color;
   text-shadow: $close-text-shadow;
-  opacity: .5;
+  // Opinionated: add "hand" cursor to non-disabled .close elements
+  cursor: pointer;
+  opacity: $close-opacity;
 
-  // Override <a>'s hover style
-  @include hover {
-    color: $close-color;
+  &:hover,
```

Comment:
We got a `hover-focus` mixin for this. Can you change this?

Reply:
@MartijnCuppens It's been deprecated at https://github.com/twbs/bootstrap/blob/v4-dev/scss/mixins/_hover.scss#L1. Let me know if you still want me to resend it.

Reply:
The `hover` mixin is deprecated, but I'm not sure about the state of the `hover-focus` mixin. @mdo, what shall we do here?

Reply:
`hover-focus` is fine, `hover` is gone.